configs.py

```diff
@@ -57,7 +57,8 @@ def parse(self, args=None):
 
     def clean_file_name(self):
         """Retrieves the clean image file_name for saving purposes"""
-        self.conf.img_name = self.conf.input_image_path.split('/')[-1].replace('ZSSR', '') \
+        path_separator = '\\' if os.name == 'nt' else '/'
```

Comment:
better to use os.sep, see:
https://docs.python.org/3.3/library/os.html?highlight=os.sep#os.sep

Reply:
cool done